backend/stakepoold/server.go

```diff
@@ -377,6 +382,27 @@ func runMain() error {
 		}()
 	}
 
+	if cfg.Prometheus {
+		var addr string
+		var wait time.Duration
+
+		if cfg.PrometheusListen == "" {
+			addr = ":9101"
```

Comment:
Default values should be applied in `config.go`. For example...

https://github.com/decred/dcrstakepool/blob/625bcd23ec3901d7b732903b9255a1a03104e0d2/config.go#L40

https://github.com/decred/dcrstakepool/blob/625bcd23ec3901d7b732903b9255a1a03104e0d2/config.go#L327

This applies for both `PrometheusListen` and `PrometheusWait`

Reply:
Thanks for suggesting this!